src/mods/mod_signup.cpp

```diff
@@ -1010,39 +1008,55 @@ bool ModSignup::birthday(const std::string &input)
         baseProcessDeliverNewLine();
 
         // Validate Date Here,  ie.. 2016-01-01 format.
-        boost::regex date_regex { m_config->regexp_date_validation };
-        boost::smatch str_matches;
-
-        if(boost::regex_match(key, str_matches, date_regex))
-        {
-            // Append Time For Date.
-            key += " 00:00:00";
-            struct std::tm tm;
-
-            tm = boost::posix_time::to_tm(boost::posix_time::time_from_string(key));
-
-            /* Only GCC 5.1 + Compatible
-            std::istringstream ss(key);
-
-            ss >> std::get_time(&tm, "%Y-%m-%d %H:%M:%S");
-            if(ss.fail())
-            {
-                std::cout << "regex passed, ss failed!" << std::endl;
-                ss.clear();
-                displayPromptAndNewLine(PROMPT_DATE_INVALID);
-                redisplayModulePrompt();
-                return true;
-            }*/
-
-            std::time_t const time = mktime(&tm);
-            m_user_record->dtBirthday = time;
-            changeNextModule();
-        }
-        else
-        {
-            displayPromptAndNewLine(PROMPT_DATE_INVALID);
-            redisplayModulePrompt();
-        }
+		std::cout << "exp: " << m_config->regexp_date_validation << std::endl;
+		try
+		{
+			std::regex date_regex { m_config->regexp_date_validation };
+			std::smatch str_matches;
+
+			if(std::regex_match(key, str_matches, date_regex))
+			{
+				// Append Time For Date.
+				key += " 00:00:00";
+				struct std::tm tm;
+
+				// Test lateron, this should work on < 5.1 gcc
+				//strptime(key, "%Y-%m-%d %H:%M:%S", &tm);
+				
+				
+				//tm = boost::posix_time::to_tm(boost::posix_time::time_from_string(key));
+
+				/**
+				 * Only GCC 5.1 + Compatible
+				 * Were no longer using boost, so bye bye 4.9 compat for now.
+				 */
+				std::istringstream ss(key);
+
+				ss >> std::get_time(&tm, "%Y-%m-%d %H:%M:%S");
```

Comment:
Note, Sticking with std::get_time which is only available in gcc 5.x.  No longer supporting older 4.x gcc compatibility.